test_script.py

```diff
@@ -0,0 +1,86 @@
+import os
+import pytest
+
+
+from bio_files_processor import FastaRecord, OpenFasta
+from ultimate_bioinf_tool import (AminoAcidSequence, DNASequence, RNASequence)

```

Comment:
```suggestion
import os
import pytest
from bio_files_processor import FastaRecord, OpenFasta
from ultimate_bioinf_tool import (AminoAcidSequence, DNASequence, RNASequence)
```

Тут всего 4 импорта, можно и не разбивать строками. Ну либо pytest надо отделить от os)